spk/minio/src/service-setup.sh

```diff
@@ -0,0 +1,46 @@
+# Setup environment
+PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
+
+CONFIG_DIR="${SYNOPKG_PKGDEST}/var"
+CFG_FILE="${CONFIG_DIR}/options.conf"
```

Comment:
May I propose to append your wizard variables to INST_VARIABLES: https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.installer#L9
which is designed to survive upgrade process and should re-inject values if wizard pops up again (when adding a new option with new version for instance)

Reply:
You mean storing my settings in the INST_VARIABLES file instead of my custom config file or in addition to that? I've also check the file on my system. It's readable by everyone. This might not be a good place to store passwords.

Reply:
Relevant point - my fault in fact. May you please add chown/chmod to restrict variables file access to only package service user so?

Reply:
Which one do you mean? ../target/var/options.cfg is only accessible by sc-minio on my system.
Maybe you should clarify what you mean so we don't talk past each other.